source/hardware/controllers/denon_mc7000.rst

```diff
@@ -100,301 +78,377 @@ Layout and functions
    Denon MC7000 (schematic view)
 
 
-1.  **Input Selector:** Set this switch to the desired audio source from
-    this channel. Channel’s controls will only send MIDI information
-    when its input selector is set to USB.
+----
```

Comment:
Why all these lines? Just use a table if you want that:

```rst
==========  ==========
Col Head 1  Col Head 2
==========  ==========
Row 1       Some text for the second column of row 1.
Row 2       Some text for the second column of row 2.
==========  ==========
```

Reply:
just wanted a separation of the single numbers. otherwise it is hard to read. Dont really need tables. would only complicate the things even more

Reply:
Let's see how it looks once you fixed the deploy preview.

Reply:
Hmm, I checked this branch out locally and removed them, to compare how it looks. IMHO, it's better without the lines. The main issue is that this is a very long list that is hard to navigate. How about splitting up the list of controls into different sections, like the Pioneer DDJ-200 mapping does?

https://manual.mixxx.org/2.3/en/hardware/controllers/pioneer_ddj_200.html

Reply:
> How about splitting up the list of controls into different sections, like the Pioneer DDJ-200 mapping does?

Could do so but a re-write will take me some time as I am busy on other things right now.

Reply:
> Please merge: [toszlanyi#1](https://github.com/toszlanyi/manual/pull/1)

Looks nice, not really shorter though but segments give a better overview - still needs a bit of tuning. Thanks a lot for your input on this and let me finish until Tuesday hopefully. Then I send another commit to this PR. Cheers!

Reply:
You can add some `=` to the rightmost header. That will make the right column wider in comparison to the other two columns.

Reply:
I like the alternating row colors but the more I use the table view the less I like it. Unfortunately the table wont get wider in the rendered site adding `=`. Also on your DJ505 site I don't like the behavior of the table that you need to move the scrollbar to the right in order to see more than 6 words. This becomes worst in case the table is longer than the page height as you need to read first 6 words then scroll down, scroll right, scroll up again to find the initial point and read the last part of the description while the first part is invisible in column 3. Maybe you scrolled too far and then you only catch the first few words and the last but not the middle of the sentence. Really annoying. It would help to define an automatic line break for columns so that longer lines break into the next one into the same column. This makes it less annoying but increases the length of the document even more.